compiler/src/middle_end/linearize.re

```diff
@@ -259,6 +259,7 @@ let rec transl_imm =
     | Left(imm) => (imm, [])
     | Right((name, cexprs)) => (Imm.id(~loc, ~env, name), cexprs)
     }
+  | TExpUse(_)
   | TExpNull => (Imm.const(~loc, ~env, Const_void), [])
```

Comment:
Why is this a void?

Reply:
All expressions in Grain produce a value, so when something "doesn't" it's void.

Reply:
Speaking of which, I don't think `PExpNull` is used anymore, so we can remove it.